src/query/graphite/native/builtin_functions.go

```diff
@@ -2406,9 +2405,7 @@ func init() {
 	MustRegisterFunction(groupByNode).WithDefaultParams(map[uint8]interface{}{
 		3: "average", // fname
 	})
-	MustRegisterFunction(groupByNodes).WithDefaultParams(map[uint8]interface{}{
-		3: nil, // nodes
-	})
+	MustRegisterFunction(groupByNodes)
```

Comment:
I think both `asPercent` and `groupByNodes` functions will need to be updated to take in a `multiplePathSpecs` instead of a `singlePathSpec` so they'd get properly cast as variadic

Reply:
Oh, they are already variadic due to `nodes ...int` third arg. 

I added a test which was returning an error before and now passes in `engine_test.go` 👍

Reply:
Ah nice, may have to remove `aliasByNode(foo.*.zed)` case from `TestCompileErrors`  then :+1: